lib/fs.js

```diff
@@ -95,7 +98,14 @@ function sanitizeFD(fd) {
 // for callbacks that are passed to the binding layer, callbacks that are
 // invoked from JS already run in the proper scope.
 function makeCallback(cb) {
-  if (typeof cb !== 'function') {
+  if (typeof cb === 'undefined') {
+    // TODO throw error instead of deprecation warning in v8.0.0
```

Comment:
I did remember the other TODO/FIXME/etc.'s format is not like this, suggest:

```js
// TODO(thefourtheye): throw error instead of deprecation warning in v8.0.0
```